inst/odin/vaccine.R

```diff
@@ -556,7 +556,7 @@ output(priorvaccinated[]) <- sum(S[i,6]) + sum(E1[i,6]) + sum(E2[i,6]) + sum(IMi
   sum(R1[i,6]) + sum(R2[i,6]) + sum(D[i,6])
 dim(priorvaccinated) <- N_age
 
```

Comment:
This is what you were talking about earlier? So dead individuals not included for the `pr[]` calculation so have to add back here. 

`# D not included in earlier total population size in order to estimate proportion of alive population vaccinated`